MAUI/ProgressBar/migration.md

```diff
@@ -0,0 +1,587 @@
+                                   TrackFill="Violet"
+                                   Progress="75">
+    <progressBar:SfCircularProgressBar.GradientStops>
+        <progressBar:ProgressGradientStop Color="#00bdaf" Value="0"/>
+        <progressBar:ProgressGradientStop Color="#2f7ecc" Value="50"/>
+    </progressBar:SfCircularProgressBar.GradientStops>
+    
+</progressBar:SfCircularProgressBar>
+
+...
+
+</ContentPage>
+ 
+{% endhighlight %}
+
+{% highlight C# %}
+
+using Syncfusion.Maui.ProgressBar;
+...
+//Snippet for the linear progress bar
+
+SfLinearProgressBar linearProgressBar = new SfLinearProgressBar();
+linearProgressBar.AnimationEasing = Easing.CubicOut;
+linearProgressBar.Progress = 75;
+linearProgressBar.SegmentGapWidth = 5;
+linearProgressBar.SegmentCount = 4;
+linearProgressBar.IndeterminateAnimationEasing = Easing.BounceIn;
+linearProgressBar.ProgressFill = Colors.Blue;
+linearProgressBar.TrackFill = Colors.Violet;
+linearProgressBar.GradientStops.Add(new ProgressGradientStop { Color = Color.FromArgb("00bdaf"), Value = 0 });
+linearProgressBar.GradientStops.Add(new ProgressGradientStop { Color = Color.FromArgb("2f7ecc"), Value = 50 });
+this.Content = linearProgressBar;
+
+//Snippet for the circular progress bar
+
+SfCircularProgressBar circularProgressBar = new SfCircularProgressBar();
+circularProgressBar.AnimationEasing = Easing.CubicOut;
+circularProgressBar.Progress = 75;
+circularProgressBar.SegmentGapWidth = 5;
+circularProgressBar.SegmentCount = 4;
+circularProgressBar.IndeterminateAnimationEasing = Easing.BounceIn;
+circularProgressBar.ProgressFill = Colors.Blue;
+circularProgressBar.TrackFill = Colors.Violet;
+circularProgressBar.GradientStops.Add(new ProgressGradientStop { Color = Color.FromArgb("00bdaf"), Value = 0 });
+circularProgressBar.GradientStops.Add(new ProgressGradientStop { Color = Color.FromArgb("2f7ecc"), Value = 50 });
+this.Content = circularProgressBar;
+...
+
+{% endhighlight %}
+
+{% endtabs %}
+</td>
+</tr>
+</table>
+
+
+## SfLinearProgressBar
+
+<table> 
+<tr>
+<th>Xamarin SfProgressBar</th>
+<th>.NET MAUI SfProgressBar</th>
+<th>Description</th></tr>
+<tr>
+<td>{{'[SecondaryProgressColor](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfLinearProgressBar.html#Syncfusion_XForms_ProgressBar_SfLinearProgressBar_SecondaryProgressColor)'| markdownify }}</td>
+<td>{{'SecondaryProgressFill'| markdownify }}</td>
+<td>Gets or sets the brush that paints the interior area of the secondary progress.</td></tr>
+<tr>
+<td>{{'[CornerRadius](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfLinearProgressBar.html#Syncfusion_XForms_ProgressBar_SfLinearProgressBar_CornerRadius)'| markdownify }}</td>
+<td>Divided into {{'TrackCornerRadius'| markdownify }}, {{'ProgressCornerRadius'| markdownify }} and {{'SecondaryProgressCornerRadius'| markdownify }}</td>
+<td>Gets or sets a value to determine the corner radius of the track, progress and secondary progress.</td></tr>
```

Comment:
<td>Gets or sets a value to determine the corner radius of the track, progress, and secondary progress.</td></tr>

Reply:
done